dev/com.ibm.ws.microprofile.reactive.messaging.kafka/src/com/ibm/ws/microprofile/reactive/messaging/kafka/KafkaInput.java

```diff
@@ -89,7 +89,9 @@ public KafkaInput(KafkaAdapterFactory kafkaAdapterFactory, KafkaConsumer<K, V> k
     }
 
     private PublisherBuilder<Message<V>> createPublisher() {
-        PublisherBuilder<Message<V>> kafkaStream = ReactiveStreams.generate(() -> 0).flatMapCompletionStage(x -> this.ackTracker.waitForAckThreshold().thenCompose(y -> pollKafkaAsync())).flatMap(Function.identity()).map(this::wrapInMessage).takeWhile((record) -> this.running);
+        PublisherBuilder<Message<V>> kafkaStream = ReactiveStreams.generate(() -> 0)
+                                                                  .flatMapCompletionStage(x -> this.ackTracker.waitForAckThreshold().thenCompose(y -> pollKafkaAsync()))
+                                                                  .flatMap(Function.identity()).map(this::wrapInMessage).takeWhile((record) -> this.running);
```

Comment:
Might look better with each operation on its own line.

Reply:
Might look better if it wasn't just one stupidly long statement!

Reply:
I mean, you can see if that looks better but I doubt it

Reply:
Done